package.json

```diff
@@ -14,6 +14,8 @@
   "homepage": "http://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
+    "budo": "budo",
```

Comment:
Is the intent here to simply expose `budo` for developers wanting to call it manually (e.g. `npm run budo -- some-args`)?